src/core/mempool.rs

```diff
@@ -425,6 +572,53 @@ impl MemPoolDB {
         query_row_columns(self.conn(), sql, args, "origin_address")
     }
 
+    /// Add estimated fee rates to the mempool rate table using
+    /// the supplied `CostMetric` and `CostEstimator`. Will update
+    /// at most `max_updates` entries in the database before returning.
+    ///
+    /// Returns `Ok(number_updated)` on success
+    pub fn estimate_tx_rates<CE: CostEstimator + ?Sized, CM: CostMetric + ?Sized>(
```

Comment:
The mempool really can grow unbound.  Is there a strategy for prioritizing transactions for fee reconsideration beyond just doing so in the order in which they were inserted into the mempool (which is what's implicitly happening in the SQL query here)?

Reply:
Nope, there isn't, we could instead sort by random (or by raw fee, though that has the potential to skew things).